crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -219,13 +220,13 @@ mod tests {
                 .unwrap();
         }
 
-        (test_network, cell_network, recv_task, rx_complete)
+        (test_network, cell_network, author, recv_task, rx_complete)
     }
 
     /// Call the workflow
-    async fn call_workflow(env: EnvWrite, cell_network: HolochainP2pCell) {
+    async fn call_workflow(env: EnvWrite, cell_network: HolochainP2pDna, author: AgentPubKey) {
```

Comment:
you had elsewhere removed the `cell_` prefix from the network name, do you want to do that here too?